### ISSUE TITLE
question

### ISSUE THREAD

**Issue by wl-stu** [ThomasNorr/QPM#1]
I tested the model on the uncropped CUB dataset, and the accuracy I obtained was around 66%. In the paper, the reported accuracy is significantly higher (around 88%).

Could you please clarify whether this discrepancy is due to the fact that I did not use the cropped version of the CUB dataset?

Thank you very much for your guidance.

**Comment by ThomasNorr** [comment]
Hi,

yes, the uploaded model is trained on the cropped dataset and hence loses performance if evaluated without the crop. 
The uploaded model reaches 85.5% on the cropped dataset, as documented in the included json file. Across 5 seeds, we got 85.1% on average (Table 2). Training a QPM without crop on CUB, gets you around 82.9% (Table 11).

Best Regards,
Thomas

**Comment by wl-stu** [comment]
Hi

Thank you for your reply. I will try training on the cropped dataset. I really appreciate your help.

Best Regards,
Wei



---- Replied Message ----
| From | Thomas ***@***.***> |
| Date | 08/27/2025 20:37 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [ThomasNorr/QPM] question (Issue #1) |
ThomasNorr left a comment (ThomasNorr/QPM#1)

Hi,

yes, the uploaded model is trained on the cropped dataset and hence loses performance if evaluated without the crop.
The uploaded model reaches 85.5% on the cropped dataset, as documented in the included json file. Across 5 seeds, we got 85.1% on average (Table 2). Training a QPM without crop on CUB, gets you around 82.9% (Table 11).

Best Regards,
Thomas

**Comment by wl-stu** [comment]
Hello,  
Does this project support resuming training from checkpoints, or do I need to implement it myself?  
Thank you for your help.



---- Replied Message ----
| From | Thomas ***@***.***> |
| Date | 08/27/2025 20:37 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [ThomasNorr/QPM] question (Issue #1) |
ThomasNorr left a comment (ThomasNorr/QPM#1)

Hi,

yes, the uploaded model is trained on the cropped dataset and hence loses performance if evaluated without the crop.
The uploaded model reaches 85.5% on the cropped dataset, as documented in the included json file. Across 5 seeds, we got 85.1% on average (Table 2). Training a QPM without crop on CUB, gets you around 82.9% (Table 11).

Best Regards,
Thomas

**Comment by ThomasNorr** [comment]
Hi,

it does not support checkpointing during the individual training runs, like the dense training or fine-tuning step. On our somewhat old 2080, the dense training only takes 3-4 hours, which is why it is not implemented.
However, the resulting models of each of these steps are saved and reused, if the seed is the same.

And to be clear: The uploaded model is trained on the cropped dataset. If you are looking for a model that works on the uncropped CUB dataset, you would need to train on that.

**Comment by ThomasNorr** [comment]
Hi,

fyi, I just uploaded a model trained on the uncropped CUB dataset. 

Best Regards,
Thomas
